persistence/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/multinode/AbstractClusteredPersistentEntitySpec.scala

```diff
@@ -141,13 +148,13 @@ abstract class AbstractClusteredPersistentEntitySpec(config: AbstractClusteredPe
 
   "A PersistentEntity in a Cluster" must {
 
-    "send commands to target entity" in within(20.seconds) {
+    "send commands to target entity" in within(75.seconds) {
       // this barrier at the beginning of the test will be run on all nodes and should be at the
       // beginning of the test to ensure it's run.
       enterBarrier("before-1")
 
       val ref1 = registry.refFor(classOf[TestEntity], "1").withAskTimeout(remaining)
-      val ref2 = registry.refFor(classOf[TestEntity], "2")
+      val ref2 = registry.refFor(classOf[TestEntity], "2").withAskTimeout(remaining)
```

Comment:
This makes the `ask-timeout` runtime-specific (dependant on the `within(duration)` above).